_i18n/zh-cn.yml

```
@@ -16,75 +16,75 @@ footer:
   documentation: 文档
   features: 特性
   web_editor: 网页编辑器
-  release_archive: 发布版归档
+  release_archive: 版本归档
   source_code: 源代码
   project: 项目
   blog: 博客
   code_of_conduct: 行为准则
   governance: 治理模式
   teams: 团队
-  priorities: 优先事项
+  priorities: 功能优先级
   communities: 社区
   resources: 资源
-  asset_library: 资产库
+  asset_library: 资源库
   press_kit: 宣传材料
-  showcase: 展示
+  showcase: 案例展示
   education: 教育
   foundation: 基金会
   about: 关于
   donate: 捐助
   license: 许可证
   privacy_policy: 隐私政策
   contact_us: 联系我们
-  text: "Juan Linietsky，Ariel Manzur 与<a href='https://github.com/godotengine/godot/blob/master/AUTHORS.md' target='_blank' rel='noopener'>贡献者们</a>。由 <a href='https://godot.foundation/' target='_blank' rel='noopener'>Godot 基金会</a>架设。网站<a href='https://github.com/godotengine/godot-website' target='_blank' rel='noopener'>源码可在 GitHub 上获取</a>."
+  text: "Juan Linietsky，Ariel Manzur 和<a href='https://github.com/godotengine/godot/blob/master/AUTHORS.md' target='_blank' rel='noopener'>其他贡献者们</a>。由 <a href='https://godot.foundation/' target='_blank' rel='noopener'>Godot 基金会</a>架设。网站<a href='https://github.com/godotengine/godot-website' target='_blank' rel='noopener'>源码可在 GitHub 上获取</a>."
 sponsors:
-  title: "Godot 由以下企业赞助："
-  text: 如果您对企业赞助感兴趣，请发邮件至 <a href='mailto:contact@godot.foundation'>contact@godot.foundation</a>。<br>作为个人捐助，你可以通过 <a href='https://fund.godotengine.org/'>Godot Engine 开发基金</a> 进行。
+  title: "Godot 的赞助者们："
```

```suggestion
  title: "感谢以下 Godot 赞助方："
```
赞助者们 sounds like a hard translation. I think it can be phrased differently while still serving the purpose of introducing the sponsors.